src/main/java/net/finmath/fouriermethod/models/BatesModel.java

```diff
@@ -94,23 +106,26 @@
 	 * @param delta Jump size variance.
 	 */
 	public BatesModel(
+			LocalDate referenceDate,
```

Comment:
Its maybe good to keep the old constructor for backward compatibility. (For example: I am not sure if the spreadsheets use this constructor?!). Its OK to put the old constructor to `@deprecated` .  Guideline is that version X.Y.(Z++) and X.Y(++).Z do not break old code. But X++ is allowed to break code and we can remove @deprecated code then.

Reply:
Fine for me, thanks!